APCAppCore/APCAppCore/Startup/APCAppDelegate.m

```diff
@@ -1146,5 +1188,57 @@ - (void)passcodeViewControllerDidFail
     self.isPasscodeShowing = YES;
 }
 
+- (void) showResetPasscodeAlert:(__unused NSNotification*)notification
+{
+    NSString* title = NSLocalizedStringWithDefaultValue(@"Reset Passcode", @"APCAppCore", APCBundle(), @"Enter Passcode", @"Prompt to change passcode");
+    NSString* message = NSLocalizedStringWithDefaultValue(@"In order to reset your passcode, you'll need to log out of the app completely and log back in using your email and password.", @"APCAppCore", APCBundle(), @"In order to reset your passcode, you'll need to log out of the app completely and log back in using your email and password.", @"description of what will happen next");
+    
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:title
+                                                                   message:message
+                                                            preferredStyle:UIAlertControllerStyleAlert];
+    
+    NSString* cancelTitle = [NSLocalizedStringWithDefaultValue(@"cancel", @"APCAppCore", APCBundle(), @"cancel", @"Action to cancel") uppercaseString];
```

Comment:
"Cancel" and not uppercase. This is inconsistent with Apple Human interface guidelines.